html-forms/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Forms</title>
+  </head>
+  <body>
+    <h1> Coding Form! </h1>
+    <form action="" method="get">
+      <label for="email">Email <br> </label>
```

Comment:
avoid using `br` tags for the most part. They are for specific text formatting, not page formatting. A div would be better to use if you need a block level element. This applies to the entire document